### DISCUSSION TITLE
[Proposal] Sign up for alpha of GitHub Issue form

### DISCUSSION THREAD

**Discussion by Andre601** [discord-jda/JDA#1539] [General]
GitHub currently has a private alpha of their upcoming Issue Forms running.
These forms are a blessing to have as they finally allow us to ask fo all the information we need without the user breaking the template or straight up replacing it with the error code alone.

You can request getting added to the alpha by filling out this form: https://forms.gle/8PtmDALhWLvVqQrFA

Here's an example of how the form could look like (Using it on one of my Repositories):
![image](https://user-images.githubusercontent.com/11576465/110358537-d2ebe480-803c-11eb-9b60-487327556efc.png)

### Example Use-Case

Issue Forms bring a lot of benefits to have:
- No longer can people just replace the issue template with copy-pasted stack traces.
- New options for dropdown, multi-selection, checkboxes, text fields, descriptions, etc.
- The default text field can be hidden (`issue_body: false`) or kept, which then appears at the bottom.

Downsides so far:
- No way to upload files directly when `issue_body` is disabled
- It's early alpha, meaning a lot of stuff can break while GitHub figures stuff out and listens to community feedback.

**Comment by DV8FromTheWorld** [comment]
I have submitted a request for access through that form.

**Reply by Andre601** [reply]
Received an answer so far?
They *should* respond within a few days.

**Reply by DV8FromTheWorld** [reply]
We have access as of 4-5 days ago, but I've not had a chance to actually do anything with that access yet unfortunately.
